auth4genai/snippets/get-started/prerequisites/async-auth.jsx

```diff
@@ -47,6 +47,22 @@ export const Prerequisites = ({
       type.
     </Step>,
 
+    <Step title="Configure Notification Channel settings">
+      Configure the notification channels (such as email or push notifications) for your Auth0 Application. Go to{" "}
+      <strong>
+        Applications &gt; [Your Application] &gt; Settings &gt; Client-Initiated Backchannel Authentication (CIBA) section &gt;
+      </strong>{" "}
+      and enable the{" "} notification channel(s) to use with the asynchronous authorization requests for this application.
+
+      To learn more about <strong>configuring notification channels</strong> and using the <strong>requested expiry</strong> setting, read the {" "}
+      <a
+        href="/ai/docs/intro/asynchronous-authorization#user-consent-and-notification-channel-selection"
+        target="_self"
+      >
+        Asynchronous Authorization
+      </a>
+      {" "} documentation.
+    </Step>,
```

Comment:
Preference here is to keep updates minimal within the quickstarts, and link back to an overview page about this overfall flow / notification logic. My thinking is this logic could evolve and seems this would be an ok place to make reference to the configuration and allow them to dive further should it be necessary.